src/concrete/ml/onnx/ops_impl.py

```diff
@@ -340,7 +340,7 @@ def numpy_sigmoid(
     Returns:
         Tuple[numpy.ndarray]: Output tensor
     """
-    return (1.0 / (1.0 + numpy.exp(-x)),)
+    return (numpy.exp(-numpy.logaddexp(0, -x)),)
```

Comment:
This improves the dynamic range handling of the sigmoid right ?

Reply:
It's more stable with large values which happens in some of our tests. I wasn't sure if was going to fuse properly but it seems that it does.